### ISSUE TITLE
[loss.py]  there is a  bug 

### ISSUE THREAD

**Issue by qq852518421** [klintan/pytorch-lanenet#31]
https://github.com/klintan/pytorch-lanenet/blob/43f1bf3c8ef6163357993d2f8869cd2be74741a3/lanenet/model/loss.py#L61-L67

**Comment by qq852518421** [comment]
we can change **embedding_i = embedding_b[seg_mask_i] ** to **embedding_i = embedding_b * seg_mask_i**.

but, the code **mean_i = torch.mean(embedding_i, dim=0)**  means ? 

we should change

```
seg_mask_i = (seg_gt_b == lane_idx) 
 if not seg_mask_i.any(): 
     continue 
 embedding_i = embedding_b[:, seg_mask_i[0]]
  
 mean_i = torch.mean(embedding_i, dim=1) 
 centroid_mean.append(mean_i)
```
maybe this is ok!

**Comment by zhangzhongyuan1996** [comment]
I tried your code ,but it is not work.Why?
